cspell-wordlist.txt

```diff
@@ -1,43 +1,31 @@
-Appflow
```

Comment:
Hmm I was trying to keep the sections separate because they are:

- Product names
- Component names
- All other words

Reply:
Hmm 🤔 so the question is if that is valuable over being able to `Cmd + .` to quick fix and add to the list. I'd argue I'm never going to open up this `txt` file to look at something other than adding a new word.

Reply:
I didn't even know it was possible to quickly add here. 😂 That's fine I am not set on it or anything.